README.md

```diff
@@ -2,197 +2,74 @@
 
```

Comment:
You removed the Troubleshooting section but didn't move it to another doc. You don't think this information is necessary? If you don't want to display it directly in the readme, I think we should have a separate doc with the content that it linked to from the readme.

Reply:
SGTM